client/src/main/java/de/rebleyama/client/RebleyamaClient.java

```diff
@@ -107,11 +108,81 @@ public boolean mouseMoved(int screenX, int screenY) {
 
     @Override
     public boolean scrolled(int amount) {
+        float effectiveViewportWidth = camera.viewportWidth * camera.zoom;
+        
+        if (((effectiveViewportWidth < 20480) || amount < 0) && ((effectiveViewportWidth > 480) || amount > 0)) {
+            camera.zoom += (0.3*amount);
+        }
+
         return false;
     }
 
 	@Override
 	public void dispose() {
 		// dispose of all the native resources
-	}
+    }
+    
+    private void handleMouseMovementInput() {
+        float mousePositionX = Gdx.input.getX(); 
+        float mousePositionY = Gdx.input.getY();
```

Comment:
Some comments about e.g. what the ifs do would really increase comprehensibility of the code.

Reply:
Added comments, although the code is self-explanatory if you even look at it for a few seconds